.github/workflows/ci.yml

```diff
@@ -98,12 +98,49 @@ jobs:
         env:
           RUSTFLAGS:
 
-  run-x86_64:
-    name: Run Hermit for Rust (x86_64)
+  run-hermit:
```

Comment:
I avoided using 'run' as the name as I thought it may be confusing to have a job with the same name as one of the Actions keywords, even if it can be told apart from the context.

Reply:
I see. Makes sense. 👍